src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -50,8 +50,6 @@
   <ItemGroup>
     <!-- Needed to break assembly conflict on StreamJsonRpc between microsoft.visualstudio.projectsystem.query and microsoft.visualstudio.languageservices -->
     <PackageReference Include="Microsoft.VisualStudio.RpcContracts" />
-    <!-- Pin MessagePack to a patched version with security vulnerable fix -->
-    <PackageReference Include="MessagePack" />
```

Comment:
Does this have dependency somewhere? where is this in another place?

Reply:
Yes, it was added recently in the host agnostic file.